apiserver/authentication/jwt/jwt.go

```diff
@@ -173,6 +176,9 @@ func (j *JWTAuthenticator) Parse(ctx context.Context, tok string) (jwt.Token, au
 }
 
 // RegisterJWKSCache sets up the token key cache and refreshes the public key.
+// The cache will repeatedly refresh the JWKS, if the key set is not available
+// initially then no error is returned to avoid blocking controller startup.
+// The refresh URL is validated earlier, see config.Validate()
 func (j *JWTAuthenticator) RegisterJWKSCache(ctx context.Context) error {
 	j.cache = jwk.NewCache(ctx)
```

Comment:
How does this work between controllers (HA setups)?

Reply:
The JWKS is in memory, each controller unit should fetch and store its own.